libraries/standard/mqtt/test/unit/iot_tests_mqtt_api.c

```diff
@@ -946,6 +961,107 @@ TEST( MQTT_Unit_API, OperationWaitTimeout )
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Test edge cases when searching for operations.
+ */
+TEST( MQTT_Unit_API, OperationFindMatch )
+{
+    int32_t i = 0;
+    uint16_t packetIdentifier = 0;
+    IotMqttError_t status = IOT_MQTT_STATUS_PENDING;
+    _mqttOperation_t * pMatchedOperation = NULL;
+    _mqttOperation_t * pOperation[ 2 ] = { NULL, NULL };
```

Comment:
This `2` also appears in both `for` loops. Should it be a macro?

Reply:
Changed